src/_pytest/assertion/util.py

```diff
@@ -96,6 +96,13 @@ def _format_lines(lines: Sequence[str]) -> List[str]:
     return result
 
 
+def _get_indent(indent_level: int) -> str:
+    """
+    Returns the indentation string based on required indentation level.
+    """
+    return " " * (indent_level * 4)
```

Comment:
It seems that `_format_lines` already uses 2-space indents we should probably be consistent and use that instead of 4-space indents.